ui/packages/consul-ui/app/components/topology-metrics/status/index.hbs

```diff
@@ -1,12 +1,16 @@
-{{#if @noMetricsReason}}
+{{#if (eq @noMetricsReason 'ingress-gateway')}}
   <span class="topology-metrics-error">
-    Unable to load metrics
+    {{t "components.consul.topology-metrics.status.ingress-gateway"}}
+  </span>
+{{else if (eq @noMetricsReason 'remote-dc')}}
+  <span class="topology-metrics-error">
+    {{t "components.consul.topology-metrics.status.error"}}
     <span>
-      <Tooltip>{{@noMetricsReason}}</Tooltip>
+      <Tooltip>{{t "components.consul.topology-metrics.status.remote-dc"}}</Tooltip>
     </span>
   </span>
 {{else if @error}}
-  <span class="topology-metrics-error">Unable to load metrics</span>
+  <span class="topology-metrics-error">{{t "components.consul.topology-metrics.status.error"}}</span>
 {{else}}
-  <span class="topology-metrics-loader">Loading Metrics</span>
+  <span class="topology-metrics-loader">{{t "components.consul.topology-metrics.status.loading"}}</span>
 {{/if}}
```

Comment:
This area feels like the conditional could be formatted better somehow? The reason I noticed this is you have the `<span class="topology-metrics-error">` repeated multiple times.

Reply:
Refactored